src/upload/src/interface.ts

```diff
@@ -73,3 +73,8 @@ export interface XhrHandlers {
 export interface UploadInst {
   submit: () => void
 }
+
+export type OnBeforeUpload = (data: {
+  file: FileList | null
```

Comment:
```suggestion
  file: FileList | null
```

FileList 是啥，不是 FileInfo?

Reply:
这边传入beforeUpload的file考虑过用fileInfo，但是翻阅了其他类似的upload组件源码，大多数都选择传原生的file对象，所以选择了fileList（upload后的原生file）。

Reply:
统一组件库的 API 是有必要保持一致性的，所以这种问题请参考 NaiveUI upload 的其他 API，而不是参考其他组件库

Reply:
你看下面的 fileList 都写了 FileInfo[]，在这里换成 FileList 就有点奇怪了，而且 file 是具体的某一个文件，所以改成 FileInfo 吧

Reply:
@amadeus711 前两天休息的时候没看到回复不好意思，学习啦，已经按其他 API 修改了。